### ISSUE TITLE
[Enhancement] Add support for using custom WIM file instead of ISO provided WIM 

### ISSUE THREAD

**Issue by bigfloppa234** [pbatard/rufus#2016]
I'd like to see this feature in Rufus (example, using a Windows 10 ISO and using a Windows 11 install.wim file to bypass the specifications check, or being able to use really any WIM file that has been customized or modified in some way)

**Comment by pbatard** [comment]
Rufus already supports selecting a `.wim`.

Have you actually tried before opening a request?

Of course, the __only__ thing you can do with a `.wim`, since will be missing the __MANDATORY UEFI AND BIOS BOOTLOADERS WHICH MUST BE PROVIDED EXTERNALLY TO BE ABLE TO CREATE A BOOTABLE USB, SINCE THEY ARE COPYRIGHTED CONTENT THAT A NON-MICROSOFT APPLICATION CANNOT LEGALLY PROVIDE ON ITS OWN__, is a Windows To Go drive...

Oh, and you will also be missing the WinPE installation environment provided in `sources\boot.wim`, so even if you have the bootloaders, you still won't have an installer.

In short, the __ONLY__ thing you can do with a `.wim` is a Windows To Go drive, which Rufus already supports. If you want to have a bootable installation media, you __will__ have to create an ISO, because Rufus cannot fill the missing content that it needs to find besides an `install.wim` and that is only to be found on an ISO.
